### ISSUE TITLE
Improve event OpenGraph metadata

### ISSUE THREAD

**Issue by samwilson** [geogeeks-au/geogeeks-au.github.io#42]
Doesn't make good preview cards at the moment, e.g. https://www.opengraph.xyz/url/https%3A%2F%2Fgeogeeks.org%2F2023%2F0705_qfield-osm.html

**Comment by VeckoTheGecko** [comment]
I don't think this is limited to events. The home site doesn't have good og metadata either https://www.opengraph.xyz/url/https%3A%2F%2Fgeogeeks.org%2F. Happy to work on this PR (mainly preview image and description), I'll assume for now just site-wide OG metadata (dynamically pulling from `_data/masthead.yml:description`) rather than page specific.

At the moment, the description is taken from the `description` metadata entry for pages, which isn't set for any of the pages at the moment.

https://github.com/geogeeks-au/geogeeks-au.github.io/blob/24eeabf2aa1e5ef4bc776e5b4507541e15832ae6/_layouts/base.html#L9

I'll also look at making the geogeeks logo the preview thumbnail.

**Comment by samwilson** [comment]
Thanks! That sounds great. `description` used to be set, but it was the full text of the event and so probably not the best for OG anyway. Perhaps for events we could just have a static description for now, e.g. "An event run by Geogeeks in Western Australia." or something simple, maybe with the date thrown in too.

We have a generic description here as well, that might be good for the homepage: https://github.com/osmlab/osm-community-index/blob/main/resources/oceania/australia/geogeeks_perth.json

**Comment by VeckoTheGecko** [comment]
Hmm. Did some looking and couldn't find a way that the `description` can be set in `events.html`. Looks like it only looks to the front matter for the page. Maybe set the `description` for all the events pages, and we can have a sitewide `if not page.description -> fallback`?

Its a bit manual, but that would probably be the easiest way to do page specific og descriptions

**Comment by BudgieInWA** [comment]
I like using the beginning of the event text for a short description. I have been composing the the event descriptions on OsmCal with that in mind, so the snippet shown when the link is posted in slack looks good.
